manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 21.8.18.1
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+Author: Microsoft
+PackageName: PIX on Windows
+PackageUrl: https://devblogs.microsoft.com/pix/download/
+License: Copyright (c) Microsoft Corporation. All rights reserved.
+ShortDescription: Performance tuning and debugging for DirectX 12 games on Windows
+Moniker: PIX
+Tags:
+- D3D12
+- DXR

```

These should be lower-case for consistent sake
```suggestion
Moniker: pix
Tags:
- d3d12
- dxr
```